config/src/config.rs

```diff
@@ -1622,7 +1622,7 @@ pub fn default_hyperlink_rules() -> Vec<hyperlink::Rule> {
         hyperlink::Rule::with_highlight(r"<(\w+://\S+)>", "$1", 1).unwrap(),
         // Then handle URLs not wrapped in brackets
         // and include terminating ), / or - characters, if any
-        hyperlink::Rule::new(r"\b\w+://\S+[)/a-zA-Z0-9-]+", "$0").unwrap(),
+        hyperlink::Rule::with_highlight(r"[^(]\b(\w+://\S+[)/a-zA-Z0-9-]+)", "$1", 1).unwrap(),
```

Comment:
Does this need to use a capturing group?  Seems like the diff could just be to prefix `[^(]` ?

Reply:
Yes, it needs. An extra character is included in a link if the capturing group is not used, like this:
```
        regex = '[^(]\\b\\w+://\\S+[)/a-zA-Z0-9-]+',
        format = '$0',
```
```
- [http://localhost/foo]
  ^^^^^^^^^^^^^^^^^^^^^ --> "[http://localhost/foo"
- http://localhost/foo
 ^^^^^^^^^^^^^^^^^^^^^ --> " http://localhost/foo"
```